resources/views/components/pulse.blade.php

```diff
@@ -50,7 +47,7 @@
             </main>
         </div>
 
-        @livewireScripts
-        @stack('scripts')
```

Comment:
The `@stack('scripts')` was a remnant from our previous iterations. We no longer use this.